incubator/elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 0.4.9
+version: 0.6.1
```

Comment:
0.5.0?

Reply:
We had a few version increments on this locally and this was the next biggest version.